saleor/graphql/meta/tests/utils.py

```diff
@@ -0,0 +1,406 @@
+from ...tests.utils import get_graphql_content
+
+PRIVATE_KEY = "private_key"
+PRIVATE_VALUE = "private_vale"
+
+PUBLIC_KEY = "key"
+PUBLIC_KEY2 = "key2"
+PUBLIC_VALUE = "value"
+PUBLIC_VALUE2 = "value2"
+
+
+def item_contains_proper_public_metadata(
+    item_from_response,
+    item,
+    item_id,
+    key=PUBLIC_KEY,
+    value=PUBLIC_VALUE,
+):
+    if item_from_response["id"] != item_id:
+        return False
+    item.refresh_from_db()
+    return item.get_value_from_metadata(key) == value
+
+
+def item_contains_multiple_proper_public_metadata(
+    item_from_response,
+    item,
+    item_id,
+    key=PUBLIC_KEY,
+    value=PUBLIC_VALUE,
+    key2=PUBLIC_KEY2,
+    value2=PUBLIC_VALUE2,
+):
+    if item_from_response["id"] != item_id:
+        return False
+    item.refresh_from_db()
+    return all(
+        [
+            item.get_value_from_metadata(key) == value,
+            item.get_value_from_metadata(key2) == value2,
+        ]
+    )
+
+
+def item_contains_proper_private_metadata(
+    item_from_response,
+    item,
+    item_id,
+    key=PRIVATE_KEY,
+    value=PRIVATE_VALUE,
+):
+    if item_from_response["id"] != item_id:
+        return False
+    item.refresh_from_db()
+    return item.get_value_from_private_metadata(key) == value
+
+
+def item_contains_multiple_proper_private_metadata(
+    item_from_response,
+    item,
+    item_id,
+    key=PUBLIC_KEY,
+    value=PUBLIC_VALUE,
+    key2=PUBLIC_KEY2,
+    value2=PUBLIC_VALUE2,
+):
+    if item_from_response["id"] != item_id:
+        return False
+    item.refresh_from_db()
+    return all(
+        [
+            item.get_value_from_private_metadata(key) == value,
+            item.get_value_from_private_metadata(key2) == value2,
+        ]
+    )
+
+
+UPDATE_PUBLIC_METADATA_MUTATION = """
```

Comment:
Why you moved all these tests to the `saleor/graphql/meta/tests/utils.py` file?

Reply:
I dont moved tests, I moved helper functions used in tests/meta and warehouse/tests/meta

Reply:
All meta tests are in `meta/tests/test_meta_queries.py` and `meta/tests/test_meta_mutations.py`. I think we shouldn't put meta test to `<module>/tests` only for Warehouse.

Reply:
I moved meta tests related to warehouses to `meta/tests`